README.md

```diff
@@ -21,6 +21,24 @@ This will store the encrypted file at `keys/production/oauth.pem.asc`.
 cat oauth.pen.asc | bay kms decrypt > oauth.pem
 ```
 
+## Elastic Cloud
+Commands for querying and interacting with the Elastic Cloud API.
+
+#### Required environment variables
+
+> [!CAUTION]
+> Variables are deployment specific - make sure the deployment you are targeting is not a production deployment.
+
+* EC_API_KEY - Generated from the deployments Kibana settings
+* EC_CLOUD_ID - Found on the deployments Elastic Cloud 'manage' page
```

Comment:
Would this be better as a flag or argument instead of an envvar?

Reply:
Because they're creds I thought it safer to keep them stored as env vars but I suppose we could set them as env vars and reference them in the argument. Am I being too cautious in not allowing a user to expose those vars directly in an arg on the cli (and therefore making it available in their shell history)?